app/views/ReportView/components/KbMatches/columnDefs.ts

```diff
@@ -63,7 +63,21 @@ const columnDefs: ColDef[] = [
     sort: 'asc',
   },
   {
-    headerName: 'Observed Variant(s)',
+    headerName: 'Known Variants',
+    colId: 'kbVariant',
+    valueGetter: (params) => {
+      const { data: { kbMatches } } = params;
+      if (kbMatches) {
+        const kbVariants = kbMatches?.map((match) => match.kbVariant).filter((kbVariant) => kbVariant !== undefined);
```

Comment:
we can probably just use `.filter` no?

`kbMatches?.filter((match) => match?.kbVariant);`

Reply:
oh yea you're right let me simplify it

Reply:
@kttkjl actually I tried it and it didn't work, we still have to extract kbVariant from each match with .map() right?

Reply:
sorry, `match.kbVariant && match.kbVariant !== undefined`, maybe the shorthand is not enough

Reply:
unless we want to keep `null, false, 0 and ''` values?

Reply:
in which case we'll use reduce

```
kbMatches?.reduce((acc, match) => {
  if (match.kbVariant !== undefined) {
    acc.push(match.kbVariant);
  }
  return acc;
}, []);
```